SPECS/libidn2/libidn2.spec

```diff
@@ -33,30 +37,29 @@ developing applications that use libidn2.
 
 %package -n idn2
 Summary:          IDNA2008 internationalized domain names conversion tool
-License:          GPLv3+
+License:          GPL-3.0-or-later
 Requires:         %{name}%{?_isa} = %{version}-%{release}
 %if 0%{?rhel} && 0%{?rhel} <= 7
-Requires(post):   /usr/bin/install-info
-Requires(preun):  /usr/bin/install-info
+Requires(post):   /sbin/install-info
+Requires(preun):  /sbin/install-info
 %endif
 
 %description -n idn2
 The idn2 package contains the idn2 command line tool for testing
 IDNA2008 conversions.
 
 %prep
-gpgv2 --keyring %{SOURCE2} %{SOURCE1} %{SOURCE0}
+%{!?el7:%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'}
```

Comment:
Ok to keep this, but ok to remove it too since it's a little redundant with the signatures file.